targets/ChibiOS/_WiFi/inventek/es_wifi_io_chibios.c

```
@@ -0,0 +1,393 @@
+    }
+
+    return ES_WIFI_STATUS_OK;
+}
+
+// Wait for a genuinely NEW rising edge on CMDDATA_RDY (i.e. the module signaling that a fresh
+// response has become ready) - NOT merely for the line to currently read high, which could just
+// as easily be leftover from an earlier, already-fully-read response. Backed by a real EXTI
+// interrupt edge counter (see CmdDataRdyEdgeCallback above), matching the original STM32Cube HAL
+// reference driver's semaphore-based wait_cmddata_rdy_rising_event() semantics exactly.
+// @param  timeout : timeout to wait for a new edge, in mS
+int wait_cmddata_rdy_rising_event(int timeout)
```

Cppcheck (2.21.0)

[style] The function 'wait_cmddata_rdy_rising_event' should have static linkage since it is not used outside of its translation unit.

(staticFunction)